code-challenges.test.js

```diff
@@ -47,21 +83,76 @@ const studyMinutesWeek2 = {
   friday: 15,
   saturday: 65
 }
+
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
 // b) Create the function that makes the test pass.
 
+// input: an object
+// output: an array of the values sorted from least to greatest.
+
+//pseudocode: create a function with one input that takes in an object, runs Object.values to extract the values into a subarray, and then returns an array sorted by length with [.sort((a, b) => a - b)], which in this configuration sorts from least to greatest.
+
+const lofiGirlApproved = (input) => {
+  let entries = Object.values(input)
+  return entries.sort((a, b) => a - b)
+}
+
+// *****Go go gadget, Built-in methods! this one was pretty easy to get running once I figured out how Object.values works, thanks for that hint! although the capital O in object threw me off for a minute. TIL all built-in classes start with a capital letter.*****
```

Comment:
This comment alone shows that you not only ACTUALLY checked the reference but you get it. Great work Simon!